ipatests/test_xmlrpc/test_krbtpolicy.py

```diff
@@ -290,7 +290,8 @@ class test_krbtpolicy(Declarative):
             command=(
                 'krbtpolicy_mod', [user1], dict(setattr=u'givenname=Pete')
             ),
-            expected=errors.ObjectclassViolation(info='attribute "givenname" not allowed'),
+            expected=errors.ObjectclassViolation(
+                info='following attributes are not allowed: givenname'),
```

Comment:
This reads awkwardly in English because it's missing the article "the following...". Perhaps "these" instead of "following":

"these attributes are not allowed:"

or

"the following attributes are not allowed: "